examples/activities/buildvrt.js

```diff
@@ -1,14 +1,15 @@
 "use strict";
 
 var fs = require('fs'),
-    path = require('path');
+    path = require('path'),
+    util = require("util");
 
 var tmp = require('tmp');
 
-var output = require("./output"),
-    shell = require("./shell");
+var output = require("../../lib/output"),
+    shell = require("../../lib/shell");
 
-module.exports = function buildVRT(files, outputUri, callback) {
+module.exports = function buildVRT(files, outputUri, options, callback) {
```

Comment:
Backward-compatibility doesn't seem like a big deal yet, and the flexibility that `options` adds is good.